.github/workflows/ode-toolbox-build.yml

```diff
@@ -0,0 +1,67 @@
+name: ODE-toolbox build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with_gsl: ["0", "1"]
+        sympy_version: ["==1.4", ""]
+
+    steps:
+      - name: Checkout ODE-toolbox code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install apt dependencies
+        run: |
+          sudo apt update
+          sudo apt install pkg-config python3-all-dev
+
+      - name: Install GSL
+        run: |
+          cd ..
+          wget http://ftp.wrz.de/pub/gnu/gsl/gsl-2.5.tar.gz
+          tar -xzf gsl-2.5.tar.gz
+          cd gsl-2.5
+          ./configure && make && sudo make install
+          cd ..
+          cd ode-toolbox
+          export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
This could be removed as I suppose it is not used later.

Reply:
I think it's necessary, it's where the GSL dynamic libraries are installed to, by default it's not on the search path.

Reply:
The `export` in one step won't be reflected in the subsequent steps unless it is in the `$GITHUB_ENV` variable. I checked the CI logs and `LD_LIBRARY_PATH` doesn't have this particular path and is set to `/opt/hostedtoolcache/Python/3.8.8/x64/lib`. Probably GSL is also installed in this path?

Reply:
Ah, OK! The superfluous lines have been removed.